opencode.json

```
@@ -8,10 +8,30 @@
     "instructions": [
-        ".ai/rules.md",
-        ".ai/commit-rules.md",
-        ".ai/instructions/*.md"
+        ".opencode/rules.md",
+        ".opencode/commit-rules.md",
+        ".opencode/instructions/*.md"
     ],
+    "permission": {
+        "bash": {
+            "go build *": "allow",
+            "hk check *": "allow",
```

Biome (2.5.0)

[error] The key hk check * was already declared.

(lint/suspicious/noDuplicateObjectKeys)